canonical/traefik-k8s-operator
# observability

This PR adds loki-logging and grafana-dashboarding integrations to the traefik charm.

## Testing Instructions
deploy cos-lite, then refresh the traefik charm with one built from these sources. Visit the grafana frontend to see the traefik dashboard with the logs.

## Release Notes
- Added loki-logging and grafana-dashboarding integrations.
